depends/packages/expat.mk

```diff
@@ -6,7 +6,7 @@ $(package)_sha256_hash=f79b8f904b749e3e0d20afeadecf8249c55b2e32d4ebb089ae378df47
 
 # -D_DEFAULT_SOURCE defines __USE_MISC, which exposes additional
 # definitions in endian.h, which are required for a working
-# endianess check in configure when building with -flto.
+# endianness check in configure when building with -flto.
```

Comment:
this one is inherited from bitcoin, fixing it here can potentially cause merge conflicts in the future

Reply:
Thank you and sorry for not noticing that. I am going to fix it upstream.